iphone/Classes/TiApp.m

```diff
@@ -799,17 +799,19 @@ - (void)URLSession:(NSURLSession *)session task:(NSURLSessionTask *)task didSend
 
 -(void)URLSession:(NSURLSession *)session dataTask:(NSURLSessionDataTask *)dataTask didReceiveData:(NSData *)data
 {
-    if (!uploadTaskResponses) {
-        uploadTaskResponses = [[[NSMutableDictionary alloc] init] retain];
+    if(!nsURLUploadTaskResponses){
+        nsURLUploadTaskResponses = [[NSMutableDictionary alloc] init];
```

Comment:
Just took care of it. Please let me know if you need me to fix something else. I'll take care of that asap.